test/functional/test-addon.js

```diff
@@ -87,24 +87,23 @@ describe("Tests verifying that the test add-on works as expected", function() {
           chosenVariation.name === "feature-passive" ||
           chosenVariation.name === "control",
       );
-    });
-
-    describe("test the library's endStudy() function", function() {
-      before(async() => {
-        await utils.executeJs.executeAsyncScriptInExtensionPageForTests(
-          driver,
-          async callback => {
-            // TODO add tests for other reasons (?)
-            await browser.study.endStudy("expired", {
-              baseUrls: ["some.url"],
-              endingName: "anEnding",
-              endingClass: "ended-positive",
-            });
-            callback();
-          },
-        );
-      });
+    });*/
 
+    describe.skip("test the library's endStudy() function", function() {
+      // before(async() => {
```

Comment:
Why check in commented-out stuff? I'd just remove it, it'll be in the commit history if you really need to dig it up. Or at least use `/* */` :)